src/main/java/org/assertj/core/error/ShouldHaveSameTime.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ * <p/>
+ * Copyright 2012-2015 the original author or authors.
+ */
+package org.assertj.core.error;
+
+import java.util.Date;
+
+/**
+ * Creates an error message indicating that an assertion that verifies that a date has same time as other date.
+ *
+ * @author Michal Kordas
+ */
+public class ShouldHaveSameTime extends BasicErrorMessageFactory {
+
+    /**
+     * Creates a new </code>{@link ShouldHaveSameTime}</code>.
+     * @param actual the actual value in the failed assertion.
+     * @param expected the expected timestamp.
+     * @return the created {@code ErrorMessageFactory}.
+     */
+    public static ErrorMessageFactory shouldHaveSameTime(Date actual, Date expected) {
+        return new ShouldHaveSameTime(actual, expected);
+    }
+
+    private ShouldHaveSameTime(Date actual, Date expected) {
+        super("%nExpecting%n <%s>%nto represent same time as:%n <%s>%nbut the time was different.%nActual time is%n "
```

Comment:
Minor change: use two spaces indentation after a new line, I know not all messages use that but it's something that I would like to, so for new error messages let's use that indentation.